tests/test_code_format.py

```diff
@@ -6,9 +6,7 @@ def test_flake8():
     """Test source code for pyFlakes and PEP8 conformance"""
     this_dir = os.path.dirname(os.path.abspath(__file__))
     source_dir = os.path.join(this_dir, '..', 'catkin_tools')
-    cmd = ['flake8', source_dir, '--count', '--max-line-length=120']
-    # work around for https://gitlab.com/pycqa/flake8/issues/179
-    cmd.extend(['--jobs', '1'])
+    cmd = ['flake8', source_dir, '--count', '--max-line-length=120', '--ignore=E722']
```

Comment:
I would suggest to rather fix the code then ignore this valuable error.

Reply:
@clalancette just revert this commit and I'll merge it without green CI. I'll fix up the travis stuff separately.

Reply:
> @clalancette just revert this commit and I'll merge it without green CI. I'll fix up the travis stuff separately.

Sounds good, will do.